scrapy/core/scraper.py

```diff
@@ -20,6 +19,9 @@
 from scrapy.utils.spider import iterate_spider_output
 
 
+QUEUE_TUPLE = Tuple[Union[Response, Failure], Request, Deferred]
```

Comment:
I would use the same letter case as other classes and types, in line with https://docs.python.org/3/library/typing.html

```suggestion
QueueTuple = Tuple[Union[Response, Failure], Request, Deferred]
```